storage-dynamodb/src/test/java/zipkin2/storage/dynamodb/DynamoDBRule.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2016-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.storage.dynamodb;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.auth.AWSStaticCredentialsProvider;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.client.builder.AwsClientBuilder;
+import com.amazonaws.retry.RetryPolicy;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsync;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBAsyncClientBuilder;
+import org.junit.rules.ExternalResource;
+import org.testcontainers.containers.GenericContainer;
+
+public class DynamoDBRule extends ExternalResource {
+
+  // Run test container: `docker run -p 8000:8000 amazon/dynamodb-local -jar DynamoDBLocal.jar -sharedDb`
+  private GenericContainer dynamoDBLocal = new GenericContainer<>("amazon/dynamodb-local")
```

Comment:
handy. not now but later we might extract the ability to run against real deal. this is great tho

Reply:
FYI LocalStack uses the same "DynamoDB Local" jar when you request DynamoDB service (some people assume they re-implemented it), in case you will need more AWS services to test

Reply:
sweet